doc/nrf/app_dev/device_guides/nrf54h/ug_nrf54h20_pm_optimization.rst

```diff
@@ -0,0 +1,197 @@
+|Active         | * ``CRACEN``       | Device driver's    | Decided by the     | The device driver        |
+|peripherals    |                    | code and data      | driver             |                          |
++---------------+--------------------+--------------------+--------------------+--------------------------+
+
+Peripherals specific to the Radio Domain
+----------------------------------------
+
+The Radio Domain does not implement the *Local Suspend to RAM* state.
+
+Optimization example
+********************
+
+This example focuses on a HID device use case (for example, a mouse)
+
+A mouse has typically two main application states:
+
+* Active, when the user is moving the mouse.
+* Idle, triggered after an inactivity timeout until an input from the user is detected.
+
+The following is an example workflow for the *Active* application state:
+
+  1. Sample input data from sensors.
+  2. Construct a HID report.
+  3. Transmit the report over the radio.
+  4. Sleep until the next period (usually, less than 1 ms).
+
+Power consumption targets
+=========================
+
+The power consumption targets for this example are the following:
+
+* Sleep currents (5 V supply):
+
+  * Application core (Suspend-to-RAM): < 10 µA
+  * Radio core (Suspend-to-Idle, cache disabled): < 10 µA
+
+* Active currents (estimates at 5 V):
+
+  * Radio core in idle: ~0.25 mA
+  * App core in idle: ~0.15 mA
+  * Core wake-up every 1 ms: +0.05 mA
+  * ADC sample every 1 ms: +0.10 mA
+  * SPI transaction every 1 ms: +0.20 mA
+  * IPC message every 1 ms: +0.15 mA
+
+Recommended Kconfig configuration
+=================================
+
+Set these Kconfigs as follows for the application running on the application core:
+
+  * :kconfig:option:`CONFIG_PM` to ``y``
+  * :kconfig:option:`CONFIG_PM_S2RAM` to ``y``
+  * :kconfig:option:`CONFIG_POWEROFF` to ``y``
+  * :kconfig:option:`CONFIG_PM_S2RAM_CUSTOM_MARKING` to ``y``
+  * :kconfig:option:`CONFIG_PM_DEVICE` to ``y``
+  * :kconfig:option:`CONFIG_PM_DEVICE_RUNTIME` to ``y``
+
+Set these Kconfigs as follows for the application running on the radio core:
+
+  * :kconfig:option:`CONFIG_PM` to ``y``
+  * :kconfig:option:`CONFIG_POWEROFF` to ``y``
+  * :kconfig:option:`CONFIG_PM_DEVICE` to ``y``
+  * :kconfig:option:`CONFIG_PM_DEVICE_RUNTIME` to ``y``
+
+You should also follow these suggestions:
+
+  * Disable all unused peripherals before entering sleep (Zephyr's API does this automatically when supported).
+  * Add `zephyr,pm-device-runtime-auto` in the DTS for all peripherals with runtime PM support.
+  * Build and program an empty image on any unused core to release shared resources.
+
+Memory and cache optimizations
+------------------------------
+
+  * Relocate frequently accessed code and data to local RAM to avoid waking global domains, especially MRAM, which uses high power.
+  * Profile L1/L2 cache usage to minimize MRAM accesses.
```

Comment:
From what I understand L2 cache can't be profiled by customer. @zycz please correct me if I'm wrong.

Reply:
can customers profile L1 cache?

Reply:
Yes, they can do that using this API: https://docs.nordicsemi.com/bundle/nrfx_3.9.0/page/drivers/cache/hal.html.

Reply:
You can add the link because I don't think we have any mention about it in our documentation.